entrypoint.sh

```diff
@@ -1,4 +1,27 @@
 #!/bin/sh -l
 
+set -e
+
+NEUTRAL_EXIT_CODE=0
+
 sh -c "/usr/local/bin/php-cs-fixer fix $*"
 
+(jq -r ".comment.body" "$GITHUB_EVENT_PATH" | grep -Fq "/php-cs-fixer")
+COMMENT_KEYWORD_FOUND=$?
+
+if [ "$COMMENT_KEYWORD_FOUND" -eq 0 ]; then
+  if [[ -z "$GITHUB_TOKEN" ]]; then
```

Comment:
Is this really needed? GitHub actions itself should take care of it, we will require it then, or?

Reply:
dont know, I can drop this check in case you prefer

Reply:
maybe its usefull when someone tries to run the action locally - out of github context?

Reply:
See https://help.github.com/en/articles/virtual-environments-for-github-actions#github_token-secret - the environment variable needs to be explicitly created, so the check is probably a good idea!

👍

Reply:
Let’s keep it